etcdmgmt/etcd-mgmt.go

```diff
@@ -33,6 +33,9 @@ var (
 	ExecName = "etcd"
 	// Configuration directory for storing etcd configuration
 	ETCDConfDir = "/var/lib/glusterd"
+	// Etcd -data-dir location. This contains WAL files and log snapshots
+	// https://coreos.com/etcd/docs/latest/admin_guide.html#data-directory
+	ETCDDataDir = "/var/lib/etcd"
```

Comment:
This is something that I've been meaning to change from a long time. I don't like that these paths have been hardcoded, and that too paths that are accessible only to root.

If you can, please change these paths to follow LocalStateDir. If no etcd paths are provided, the etcd conf and data should be inside LocalStateDir. If the user requires they can provide an alternative path.

When GD2 packages are built later, the correct paths for the respective environment will be set during the build process.